src/settings/slider-setting/slider-setting.ts

```diff
@@ -0,0 +1,70 @@
+import {UIPSetting} from '../setting/setting';
+import {attr} from '@exadel/esl/modules/esl-base-element/core';
+import {WARNING_MSG} from '../../utils/warning-msg';
+
+export class UIPSliderSetting extends UIPSetting {
+  public static is = 'uip-slider-setting';
+  /** CSS Class added when setting has inconsistent state. */
+  public static inconsistencyClass = 'inconsistency-marker';
+
+  /** Setting's visible name. */
+  @attr({defaultValue: ''}) public label: string;
+  /** Minimum range value. */
+  @attr({defaultValue: '0'}) public min: string;
+  /** Maximum range value. */
+  @attr({defaultValue: '100'}) public max: string;
+  /** Step for range. */
+  @attr({defaultValue: '1'}) public step: string;
```

Comment:
It sounds like an '0' will be a more expectable value here

Reply:
Just in case: it sounds like I miss rid that it is a step but not a current value.